desc/equilibrium/equilibrium.py

```diff
@@ -167,6 +168,7 @@ def __init__(
         spectral_indexing=None,
         **kwargs,
     ):
+        super().__init__()
```

Comment:
the init method of `Optimizable` does not seem to do much, what is this call here for?

Reply:
I think this was left over from another change, will remove.